client-react/src/app/store.ts

```diff
@@ -1,9 +1,11 @@
 import { configureStore, ThunkAction, Action, combineReducers, getDefaultMiddleware } from '@reduxjs/toolkit';
+import { todosReducer } from '../features/todos/store';
 import { dataReducer } from '../store/data-reducer';
 import storeLogger from './middlewares/store-logger';
 
 const rootReducer = combineReducers({
-  data: dataReducer
+  data: dataReducer,
+  todos: todosReducer
```

Comment:
It would make sense to have here a reducer called featuresReducer that would represent the combination of all the feature reducers. Like featuresReducer = combineReducers({todos: todosReducer ...})

Reply:
Done